### PR TITLE
Updated packages section description.

### PR DESCRIPTION
**Pull request by MMaiero**
The message will now highlight the fact that a page reload may be needed
due to the asynchronous startup of bundles.

Signed-off-by: Maiero <matteo.maiero@eurotech.com>

### CODE REVIEW

**Comment by MMaiero** [comment]
@mstankovic @salvatore-coppola Can you please have a look and see if the message is clear enough? Thanks.

File: kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties [hunk 1]
```diff
@@ -132,7 +132,7 @@ packagesMarketplaceInstallConfirmMessage=Install deployment package from Eclipse
 packagesDropzoneMessage=Drop the package to install
 packagesMarketplaceInstallDpNotValid=File or link not valid
 packagesTooManyDpFiles=Too many files selected. Only one file is allowed at a time.
-packagesIntro=Manage the Deployment Packages installed on the system
+packagesIntro=Manage the Deployment Packages installed on the system. Due to the asynchronous startup of the bundles contained in a deployment package, a page reload may be required.
```
Comment (salvatore-coppola) [inline kura/org.eclipse.kura.web2/src/main/resources/org/eclipse/kura/web/client/messages/Messages.properties]: I wuold add some words highlightning one of the effect is that the services name will not be displayed in the service list section. So the refresh will solve the missing.

**Comment by MMaiero** [comment]
@salvatore-coppola Is the PR ok now?

**Comment by salvatore-coppola** [comment]
> 
> 
> @salvatore-coppola Is the PR ok now?

yes!

**Review by salvatore-coppola — APPROVED** [review]

**Pull request merged**